core/src/main/scala/org/apache/pekko/persistence/jdbc/testkit/internal/SchemaType.scala

```diff
@@ -26,6 +26,11 @@ import org.apache.pekko.annotation.InternalApi
  */
 @InternalApi private[jdbc] case object Postgres extends SchemaType
 
+/**
+ * INTERNAL API
+ */
+@InternalApi private[jdbc] case object PostgresPekkoSchema extends SchemaType
```

Comment:
Again, I don't know why we need a new schema type, I would prefer to fix the existing Postgres one.